src/typical.jl

```diff
@@ -76,8 +83,22 @@ _replace(matrix_term::MatrixTerm, typicals::Dict) = MatrixTerm(_replace.(matrix_
 _replace(term::AbstractTerm, typicals::Dict) = haskey(typicals, term) ? typicals[term] : term
 _replace(term::InteractionTerm, typicals::Dict) = InteractionTerm(_replace.(term.terms, Ref(typicals)))
 
+_rmliterals(s) = filter(x -> !isa(x, Number), s)
+_termsyms(t) = StatsModels.termsyms(t)
+_termsyms(t::FunctionTerm) = _rmliterals(union(_termsyms.(t.args_parsed)...))
```

Comment:
I wouldn't use `args_parsed`, just `args`.  `args_parsed` is there solely because we currently do things like distributive rule, associative rule, etc. at parse time, so we need to keep a record of what WOULD happen if the function term is later treated as special syntax.  As long as the `FunctionTerm` stays as a `FunctionTerm` its `args` that you want.

Reply:
Um, 

```julia
type FunctionTerm has no field args
```

Reply:
On StatsModels v0.6.25

Reply:
oh it's probably stored on the type parameter :facepalm: .  Try 

```
 _termsyms(t::FunctionTerm{Fo,Fa,Names}) where {Fo, Fa, Names} = Names
```

(Should be a tuple of symbols)